contents/tutorials/ghost-analytics.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Quick start: Ghost analytics and session replay"
+date: 2024-10-11
+featuredTutorial: false
+author:
+  - danilo-campos
+tags:
+  - product analytics
+  - web analytics
+  - session replay
+---
+
+import Snippet from "../docs/integrate/snippet.mdx"
+
+With all the drama in the world of WordPress, it’s a great time to explore Ghost, a robust blogging and newsletter CMS. You can [self-host](https://ghost.org/docs/install/) Ghost on your own server, or lean on managed hosting from [Ghost.org](https://ghost.org).
+
+However you host, adding PostHog for [web analytics](/docs/web-analytics), [product analytics](/docs/product-analytics), and [session replay](/docs/session-replay) is a quick process that gives you immediate insight about how people are finding your site, enjoying your writing and exploring your archives. PostHog accounts are free, so sign up and follow along.
+
+## Adding PostHog using Ghost’s code injection
+
+Grab the PostHog snippet from [your project settings](https://us.posthog.com/project/2/settings/project#snippet):
+
+<Snippet />
+
+If you'd like to get person profiles for every visitor, update the value for `person_profiles` to `always`.
+
+In your Ghost publication’s settings, click the Settings button, labeled with a **gear icon**:
+
+![Ghost settings menu](https://res.cloudinary.com/dmukukwp6/image/upload/ghost_settings_e1f6c4b436.png)
+
+Scroll toward the bottom of the sidebar, looking for **Advanced**, and select **Code Injection**. Open the code injection setting:
+
+![Code injection menu in Ghost](https://res.cloudinary.com/dmukukwp6/image/upload/code_injection_menu_66fa301caf.png)
+
+Paste the PostHog script into the **Site header** code block:
+
+![Ghost code injection settings](https://res.cloudinary.com/dmukukwp6/image/upload/injection_field_87c4475be8.png)
+
+Click **Save**, and you’re all set. Explore your blog to generate some activity, then have a look around your PostHog account to see the results.
```

Comment:
Do you need to publish the site again?
```suggestion
Click **Save**, and you’re all set. Explore your blog to generate some activity, then head to PostHog to see the results.
```

Reply:
Seemingly not, Ghost seems to handle that under the hood for you as changes are made.